### PR TITLE
PR: Add instance pattern "cls" for the class method

### PR DESCRIPTION
**Pull request by robert780612**
The word "cls" usually use in the class method.
I thought we should highlight "cls" just like "self".

For example
```
class Foo:
    @classmethod
    def clsmethod(cls, x):
        pass
```

### CODE REVIEW

File: spyder/utils/syntaxhighlighters.py [hunk 1]
```diff
@@ -332,6 +332,7 @@ def make_python_patterns(additional_keywords=[], additional_builtins=[]):
     builtin = r"([^.'\"\\#]\b|^)" + any("builtin", builtinlist) + r"\b"
     comment = any("comment", [r"#[^\n]*"])
     instance = any("instance", [r"\bself\b",
+				r"\bcls\b",
```
Comment (CAM-Gerlach) [inline spyder/utils/syntaxhighlighters.py]: Fix your alignment; should be one space to the right.

**Comment by ccordoba12** [comment]
@robert780612, thanks for your contribution! Since you made this PR against master, it'll be part of Spyder 4, which will be released in a year, perhaps less.

If you want this to be part of our next release (3.2.9), please move this PR to our 3.x branch by using the commands mentioned here:

https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md#changing-the-base-branch

File: spyder/utils/syntaxhighlighters.py (hunk 1 above)
Comment (robert780612) [reply spyder/utils/syntaxhighlighters.py]: I can't find any test case for the function "make_python_patterns".
Should I add a new test in the "test_syntaxhighlighters.py"?

https://github.com/spyder-ide/spyder/blob/master/spyder/utils/tests/test_syntaxhighlighters.py

Comment (CAM-Gerlach) [reply spyder/utils/syntaxhighlighters.py]: Sure, if you don't mind.

**Comment by robert780612** [comment]
@ccordoba12, I still fail to rebase this PR from master to 3.x after hours trial.
So I open another pull request for branch 3.x.
Is it ok? 
If not, just close my new pull request, thanks.

**Comment by ccordoba12** [comment]
Closing in favor of #6826.

**Pull request closed without merging**